internal/service/federatedsettingsorgconfig/data_source_federated_settings_connected_orgs.go

```diff
@@ -182,8 +181,8 @@ func flattenFederatedSettingsOrganizationConfigs(federatedSettingsConnectedOrgan
 					"identity_provider_id":       federatedSettingsConnectedOrganizations.Results[i].IdentityProviderID,
 					"org_id":                     federatedSettingsConnectedOrganizations.Results[i].OrgID,
 					"post_auth_role_grants":      federatedSettingsConnectedOrganizations.Results[i].PostAuthRoleGrants,
-					"role_mappings":              federatedsettingsidentityprovider.FlattenRoleMappings(federatedSettingsConnectedOrganizations.Results[i].RoleMappings),
-					"user_conflicts":             federatedsettingsidentityprovider.FlattenUserConflicts(*federatedSettingsConnectedOrganizations.Results[i].UserConflicts),
+					"role_mappings":              FlattenRoleMappings(federatedSettingsConnectedOrganizations.Results[i].RoleMappings),
```

Comment:
same, GetResults() better?

Reply:
Same as my last comment, this change was only to move methods from federated_settings_identity_provider to federated_settings_connected_org